CGPs/cgp-0111.md

```diff
@@ -0,0 +1,134 @@
+
+*Seamless on- and off-ramping:* MiniPay via a variety of local partners provides the ability to users to seamlessly on- and off-ramp to local currency, eg from their mobile money/airtime account, at attractive buy/sell rates, minimal fees, to lower the friction to use web3 dApps built on Celo
+
+*Readability:* Actions executed on smart contracts will be presented in a human-readable way in clear and straightforward language.
+
+*Discoverability:* Integrations will be showcased inside the curated Discover dApp store section, accessible right from the main view and offer promotional abilities to partners.
+
+*Security:* MiniPay leverages automatic Google backup for secret recovery phrases making sure users’ accounts are secured, including assets used on Discovery dApps
+
+*Storytelling:* Explaining the value proposition quickly in a way users can understand is often forgotten. We will allow developers to preface their application with a multi-screen story to quickly and visually explain what they are all about.
+
+
+## Allocation
+
+Opera is requesting $568,182 per quarter in CELO representing a joint investment to scale the Celo ecosystem across key African markets, to be used for the following:
+
+Incremental marketing budget to advertise Celo ecosystem products using relevant marketing inventory available on Opera Mini, such as display advertising, bookmark links, push & in-app notifications during onboarding of new users and the lifecycle of existing users as well as other relevant inventory to drive activations and usage for MiniPay (in addition to ongoing promotions of Minipay product itself)
+Marketing campaigns and promotional activations to drive user engagement and drive adoption.
+Dedicated business development point of contact for the Celo ecosystem and regular office hours for builders with Opera product and engineering leads; regular onboarding, engagement and integration of ecosystem partner projects into MiniPay.
+Creation of educational material to support the development of the Celo ecosystem and MiniPay. These resources will be shared through our websites and applications, including our very popular FeedNews content platform, social media channels, and community forums. Strengthening audience awareness of the product.
+Commitment to provide quarterly updates on the progress of MiniPay and its engagement with Celo ecosystem partners. This could include the Signal Celo Community Newsletter or the Celo Foundation Prosper newsletter.
+
+As mentioned before, the grant shall be paid quarterly in $CELO with Opera intending to hold, stake and the ability to participate actively in governance. Opera is also exploring joining Google, Deutsche Telekom and others in running validators for the Celo network. Opera already holds Celo and intends to continue to participate in the governance process.
+
+The receiving address is 0x6f62E32cae54D66e234F0d1e27be7e33896Dc4CE
```

Comment:
This should be a multisig address and if it is, you should provide details of the signers and each of their addresses/

Reply:
The Opera team shall withdraw funds to the listed hw wallet and then immediately route the funds to their secure.